src/components/generate.js

```diff
@@ -0,0 +1,51 @@
+import TemplateStyleSheet from '../lib/template-stylesheet.json'
+
+let GENERATED_FILE = null
+
+/**
+ * Reads the values from local storage
+ * Appends css attribute
+ * @returns array: data that can be converted to css file.
+ */
+const getData = () => {
+  const data = []
```

Comment:
no need to declare a variable like this and then push to it, instead do something like
```
return [
	':root {',
	'etceetc',
	...Object.keys(fssfsf).map()
	'}'
]
```